discovery-infra/bm_inventory_api.py

```diff
@@ -149,13 +149,13 @@ def install_cluster(self, cluster_id):
         return self.client.install_cluster(cluster_id=cluster_id)
 
 
-def create_client(inventory_url=None, wait_for_url=True):
+def create_client(namespace, inventory_url=None, wait_for_url=True):
```

Comment:
`create_client` parameters order is `(namespace, service, ...)` while `utils.get_service_url_with_retries` and `utils.get_service_url` is the opposite.
Please choose one order.

Reply:
Why this is relevant? I would understand the logic if they were received the same arguments, but they aren't related in any way. The reason create client receive namespace at the 1st arg is because the other have defaults, and this doesn't. I don't think we need to align all functions receive namespace to be at 1st place cause of that.

Reply:
i thikn we can remove defaults but as u wish